File: src/main/java/com/animetv/reactiveanimelist/domain/show/Show.java
package com.animetv.reactiveanimelist.domain.show;

import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Table;

@Table("shows")
public record Show(
    @Id String slug,
    String title,
    String description,
    String thumbnail
) {}

File: animelist-client/src/app/show/entity/show.ts
export type Show = {
  slug: string,
  title: string,
  thumbnail: string,
  description: string,
  episodes: number,
  length: number,
};

export type ShowDetail = {
  slug: string,
  title: string,
  thumbnail: string,
  description: string,
  totalEpisodes: number,
  episodes: any[],
}
